zetteldeft.el

```diff
@@ -6,7 +6,7 @@
 ;; URL: https://efls.github.io/zetteldeft/
 ;; Keywords: deft zettelkasten zetteldeft wp files
 ;; Version: 0.2
-;; Package-Requires: ((emacs "25.1") (deft "0.8"))
+;; Package-Requires: ((emacs "25.1") (deft "0.8") (dash "2.17.0"))
```

Comment:
I'm all for using `dash`. That said, is adding a dependency acceptable?

Reply:
I used `dash` to get `-reduce-from`. Is there an alternative I could use to avoid the dependency?

Reply:
Packages `cl-lib` and `seq` both have reduce functions, I believe, and are included with emacs.

If there are consistent functional patterns in the code, `dash` is a great choice. If it's just a one-off usage, I'd probably lean toward not adding dependencies.